libs/constellation/src/constellation.cpp

```diff
@@ -91,6 +90,35 @@ constexpr char const *LOGGING_NAME = "constellation";
 const std::size_t HTTP_THREADS{4};
 char const *      GENESIS_FILENAME = "genesis_file.json";
 
+class Defer
+{
+public:
+  using Callback = void (Constellation::*)();
+
+  explicit Defer(Constellation *instance, Callback callback)
```

Comment:
no need for explicit here

Reply:
Good spot - it was a single parameter function at one point. I think I will fix this on delta if you don't mind